linux-cmd-intallation.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Error handling
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
+terminate()
+{
+    echo >&2 '
+*** terminated ***
+'
+    echo "\"$last_command\" command failed with exit code $?."
+    exit 1
+}
+
+trap 'terminate' 0
+
+set -e
+
+# install required packages
+echo 'Installing package dependencies...'
+sudo dnf install python3 ruby subversion rpm-build
+
+echo 'Installing ruby gem dependencies...'
+sudo gem install asciidoctor concurrent-ruby haml tilt
```

Comment:
smth here isn't quite right, I get a deprecation message

`DEPRECATION: Configuring installation scheme with distutils config files is deprecated and will no longer work in the near future. If you are using a Homebrew or Linuxbrew Python, please see discussion at https://github.com/Homebrew/homebrew-core/issues/76621
Requirement already satisfie`